internal/functions/deploy/deploy_test.go

```diff
@@ -22,25 +21,6 @@ import (
 	"gopkg.in/h2non/gock.v1"
 )
 
-func TestMain(m *testing.M) {
```

Comment:
There are test failures that were previously hidden by an empty `TestMain` block. Specifically

```
--- FAIL: TestDeployCommand (0.00s)
    --- FAIL: TestDeployCommand/deploys_multiple_functions (0.00s)
        /Users/qiao/work/cli/internal/functions/deploy/deploy_test.go:220: 
            	Error Trace:	/Users/qiao/work/cli/internal/functions/deploy/deploy_test.go:220
            	Error:      	Received unexpected error:
            	            	open /Users/qiao/work/cli/internal/functions/deploy/supabase/functions/import_map.json: file does not exist
            	Test:       	TestDeployCommand/deploys_multiple_functions
```